nixos/modules/programs/nist-feed.nix

```
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.nist-feed;
+in
+{
+  options = {
+    programs.nist-feed = {
+      enable = mkEnableOption (mdDoc "NIST Feed notifies you about the newest published CVEs.");
+      package = mkPackageOption pkgs "nist-feed" { };
+      arguments = mkOption {
+        type = types.str;
+        default = "-l -s CRITICAL";
+        description = mdDoc ''lib.
+          Arguments to provide to NIST-Feed.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
+    systemd.user.services.nist-feed = {
+        wantedBy = [ "default.target" ];
+        description = "NIST-Feed service";
```

Having a bit more descriptive description here, saying roughly what it does would be nice.

```suggestion
        description = "A notification daemon for CVEs";
```